README.md

```diff
@@ -84,12 +84,12 @@ Is your Git repository bursting at the seams?
 
     No options are required. You can learn about available options by typing `git sizer -h` or by reading on.
 
-    (The above command assumes that you have added `git-sizer` to your `PATH`. If you don't add it to your `PATH`, you need to type its full path and filename to run it; e.g., `/path/to/bin/git-sizer`.)
+    *Note*: The above command assumes that you have added `git-sizer` to your `$PATH`. If you don't add it to your `PATH`, you need to type its full path and filename to run it; e.g., `/path/to/bin/git-sizer`. From now on, we will use `git sizer` which is interchangeable with `git-sizer` anywhere (assuming you added `git-sizer` to your `$PATH` already, which is recommended).
```

Comment:
I think that changing `PATH` to `$PATH` will make it more confusing to Windows users, since the `$` isn't used on that OS.

The proposed new sentence feels repetitive. What would you think about replacing the whole thing with the following?

> *Note*: The commands shown in this README assume that you have put the `git-sizer` executable somewhere in your `PATH`. If you don't add it to your `PATH`, you need to type its full path and filename to run it; e.g., `/path/to/bin/git-sizer`.

Reply:
Cross platform support is a good point, didn’t come to my mostly-Linux mind :)

I missed that point as I skimmed through the long READMe. And I suspect many others will. So a little repetition (like in slides) doesn’t hurt, if it’s helping the users.